src/sentry/integrations/github/client.py

```diff
@@ -319,15 +319,19 @@ def get_blame_for_file(
             path="/graphql",
             data={"query": query},
```

Comment:
One thing I've noticed is that we're trying to apply suspect commits on stacktraces that have not had code mappings applied.
<img width="331" alt="image" src="https://user-images.githubusercontent.com/44410/199010743-2fa3b850-99e8-4932-a3d4-af07c6aad731.png">
<img width="458" alt="image" src="https://user-images.githubusercontent.com/44410/199010802-c7264f92-d65a-4233-a06b-526365b5e9d5.png">